scripts/build_centos.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -e
+
+#based on
+#https://sourceforge.net/p/qwwad/wiki/Installing%20from%20a%20package/
+
+# This script builds qwwad on the centos 6 platform
+# This successfully builds in a docker image with all of the
+# following dependencies installed
+# To be run from the project root
+
+yum install -y epel-release
+yum update -y
+yum install -y gcc gcc-c++ gcc-gfortran help2man armadillo-devel boost148-program-options boost148-devel gsl-devel lapack-devel libxml++-devel cmake3 git wget
```

Comment:
Installing dependencies is a job for the system administrator.  We should really create our own RPM package instead... or just list the dependencies in the documentation